tests/model_hub_tests/models_hub_common/constants.py

```diff
@@ -11,5 +11,11 @@
     tf_hub_cache_dir = os.path.join(tempfile.gettempdir(), "tfhub_modules")
     os.environ['TFHUB_CACHE_DIR'] = tf_hub_cache_dir
 
+if os.environ.get('HUGGINGFACE_HUB_CACHE') is not None:
+    hf_hub_cache_dir = os.environ['HUGGINGFACE_HUB_CACHE']
+else:
+    hf_hub_cache_dir = tempfile.gettempdir()
+    os.environ['HUGGINGFACE_HUB_CACHE'] = hf_hub_cache_dir
```

Comment:
```suggestion
hf_hub_cache_dir = os.environ.get('HUGGINGFACE_HUB_CACHE', tempfile.gettempdir())
```

Reply:
Please do the same for other lines here.

Reply:
Suggested code will not set env var value to temporary dir

Reply:
But we don't need to set env var here because we use `hf_hub_cache_dir` entire the code

Reply:
```suggestion
hf_hub_cache_dir = os.environ.get('HUGGINGFACE_HUB_CACHE', tempfile.gettempdir())
os.environ['HUGGINGFACE_HUB_CACHE'] = hf_hub_cache_dir
```